### DISCUSSION TITLE
Considering use of langchain4j

### DISCUSSION THREAD

**Discussion by aelashry777** [quarkiverse/quarkus-langchain4j#1069] [General]
Hello, 

I am considering using langchain4j for my Java apps, but interested in what everyone’s experience has been so far. 

- Why did you choose langchain4j over other frameworks/libraries? 
- How has the experience been? What challenges have you faced? Did you later find that another framework is a better fit for one of your use cases?
- Also, are you finding that companies in general that are interested in AI app development are looking to use Java or other languages (i.e. Python)? 

Any advice, tips, or recommendations for someone generally exploring this space would be appreciated. Thank you.

**Comment by geoand** [comment]
👋🏽 

> Why did you choose langchain4j over other frameworks/libraries?

What alternatives do you have in mind here?

> Also, are you finding that companies in general that are interested in AI app development are looking to use Java or other languages (i.e. Python)?

We have seen that lots of companies are interested in figuring out how to incorporate AI into the current workflows.
